src/features/WriteStoriesModal/WriteStoriesModal.tsx

```diff
@@ -2,27 +2,272 @@ import React from "react";
+
+  const buttonPath = useGetButtonPath();
+  const doesWarningButtonExist = useGetWarningButtonStatus(
+    step === "customStory",
+    api,
+    addonsStore
+  );
+  const backdropBoundary = useGetBackdropBoundary(
+    "syntax-highlighter-backdrop",
+    step === "customStory"
+  );
+
+  const project = useGetProject();
+  const isJavascript = project?.data?.language === "javascript";
+
+  const data = isJavascript
+    ? dataJavascript
+    : project?.data?.framework.name === "@storybook/nextjs"
+    ? dataTypescriptNextjs
+    : dataTypescript;
+
+  const copyWarningStory = () => {
+    const warningContent = data[4][0].content;
+    navigator.clipboard.writeText(
+      warningContent.replace("// Copy the code below", "")
+    );
+    setWarningStoryCopied(true);
+  };
 
-export function WriteStoriesModal({ onFinish }: { onFinish: () => void }) {
   return (
-    <Modal width={540} defaultOpen>
-      {({ Title, Description }) => (
-        <ModalContentWrapper data-chromatic="ignore">
-          <StorybookLogo />
-          <Title style={{ marginTop: 20 }}>
-            Create your first story (WORK IN PROGRESS)
-          </Title>
-          <Description>
-            Now it's your turn. <br />
-            See how easy it is to create your first story by following these
-            steps below.
-          </Description>
-          <Button style={{ marginTop: 4 }} onClick={onFinish}>
-            Go to story
-          </Button>
-        </ModalContentWrapper>
+    <Modal width={738} defaultOpen>
+      {({ Title, Description: DefaultDescription, Close }) => (
+        <ModalContent>
+          <div style={{ maxHeight: "450px" }}>
+            {data ? (
+              <SyntaxHighlighter
+                activeStep={
+                  step === "imports"
+                    ? 1
+                    : step === "meta"
+                    ? 2
+                    : step === "story"
+                    ? 3
+                    : step === "args"
+                    ? 4
+                    : step === "customStory"
+                    ? 5
+                    : 1
+                }
+                contents={data}
+                width="445px"
+              />
+            ) : (
+              <div>loading...</div>
+            )}
+            {backdropBoundary && !doesWarningButtonExist?.data && (
+              <Button
+                onClick={() => {
+                  copyWarningStory();
+                }}
+                style={{
+                  position: "absolute",
+                  top: backdropBoundary.top + backdropBoundary.height - 45,
+                  left:
+                    backdropBoundary.left +
+                    backdropBoundary.width -
+                    (isWarningStoryCopied ? 115 : 100),
+                  zIndex: 1000,
+                }}
+              >
+                {isWarningStoryCopied ? <>Code copied </> : "Copy code"}
```

Comment:
I don't know if there is enough space, but "Copied to clipboard" would be more accessible to users:

```suggestion
                {isWarningStoryCopied ? "Copied to clipboard" : "Copy code"}
```